test/cli/errors-skip-cache/test.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -euo pipefail
+
+dir=$(mktemp -d)
+cleanup() {
+    rm -r "$dir"
+}
+trap cleanup EXIT
+
+mkdir "$dir/cache"
+
+run_sorbet() {
+  set -x
```

Comment:
Why not have this at the top-level?

Reply:
Ah, I was using that to debug, and should not have included it. Thank you for catching that!